test/integration/taxon_browsing_test.rb

```diff
@@ -43,6 +42,23 @@ class TaxonBrowsingTest < ActionDispatch::IntegrationTest
     then_i_can_see_navigation_to_brexit_pages
   end
 
+  it 'renders an in-page nav in variant C' do
```

Comment:
Super pedantic I know but it might be confusing to future people that is says something about variant c so I'd remove that

Reply:
fixed